rclcpp/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ find_package(rcl_interfaces REQUIRED)
 find_package(rcl_yaml_param_parser REQUIRED)
 find_package(rcpputils REQUIRED)
 find_package(rmw REQUIRED)
```

Comment:
Yeah, I agree that it should depend on `rmw`, at least due to this line, but also as @dirk-thomas said since it uses rmw types and symbols in places directly.